gke-to-gke-vpn/cleanup.sh

```diff
@@ -93,18 +93,33 @@ if deployment_exists "${PROJECT_ID}" "cluster-deployment"; then
 fi
 
 ### Delete VPN connections
-for (( c=1; c<=4; c++ ))
-do
-  if deployment_exists "${PROJECT_ID}" "vpn$c-deployment"; then
-    gcloud deployment-manager deployments delete "vpn$c-deployment" --quiet
+timeout=100
+vpn_exists=0
+
+while [ $timeout -gt 0 ]; do
+  for (( c=1; c<=4; c++ ))
+  do
+    if deployment_exists "${PROJECT_ID}" "vpn$c-deployment"; then
+      vpn_exists=$((vpn_exists+1))
+      gcloud deployment-manager deployments delete "vpn$c-deployment" --quiet
+    fi
+  done
+  if [[ "$vpn_exists" == "0" ]]; then
+    break
   fi
+  sleep 20
+  timeout=$((timeout - 20))
 done
 
+echo "All VPN's have been deleted"
+
 ### Delete static ips
 if deployment_exists "${PROJECT_ID}" "static-ip-deployment"; then
```

Comment:
So instead of this step, sleep, then the next step, we just call a function like `delete_deployment 'static-ip-deployment'` followed by `delete_deployment 'network-deployment'` and that `delete_deployment` function takes care of checking exists, sleeping, retrying, error handling, etc?